jwst/cube_skymatch/skycube.py

```diff
@@ -417,6 +417,6 @@ def combine_with_other(self, other):
         w1 = self.cube_weight * self.weights[m]
         w2 = other.cube_weight * other.weights[m]
         tw = w1 + w2
-        d = (w1 * self.data[m] + w2 * other.skysub_data[m]) / tw
+        # d = (w1 * self.data[m] + w2 * other.skysub_data[m]) / tw
```

Comment:
`d` is not used. If it is not needed it should be removed from code.

Reply:
Agree, it seems that `self._data[m] = (w1 * self.data[m] + w2 * other.skysub_data[m]) / tw` is doing the same thing. I don't remember, but maybe I was doing something like self._data[m] = d` in the past (or when developing/debugging).